beacon-chain/sync/options.go

```diff
@@ -161,3 +162,11 @@ func WithBlobStorage(b *filesystem.BlobStorage) Option {
 		return nil
 	}
 }
+
+// WithVerifier gives the sync package direct access to the verifier.
+func WithVerifier(v *verification.Initializer) Option {
```

Comment:
replace this with `*verification.InitializerWaiter` and `WaitForInitializer` in `sync.Service.Start` (right after `WaitForClock`)